docs/howtoguides/enable_anbox.md

```diff
@@ -0,0 +1,45 @@
+# How to enable Anbox Cloud
+
+```{important}
+[Anbox Cloud is supported on 20.04 and 22.04 releases](https://anbox-cloud.io/).
+```
+
+To use Anbox, you will need to enable it directly throught the Ubuntu Pro Client,
+which will install all of the necessary snaps and setup the APT sources needed for
+the service.
+
```

Comment:
In our other guides, we explain how to update Pro Client. Should we do that here as well, or does it not matter what version they use for Anbox?

Reply:
That is a good idea, I will add that section